packages/template-retail-react-app/app/components/image-gallery/index.jsx

```diff
@@ -97,6 +97,7 @@ const ImageGallery = ({imageGroups = [], selectedVariationAttributes = {}, size}
 
     const heroImage = heroImageGroup?.images?.[selectedIndex]
     const thumbnailImages = thumbnailImageGroup?.images || []
+    const isLazyLoad = lazy ? 'lazy' : 'eager'
```

Comment:
Variable names starting with 'is' are usually boolean values. So perhaps like this instead:
```suggestion
    const loadingStrategy = lazy ? 'lazy' : 'eager'
```